### PR TITLE
Fix #412 : Remove release warning message from dashboards

### LINKED ISSUE #412: Remove release warning message from dashboards
After #399 is fixed, we can remove the top-level warning message about release data:
message

### PR DESCRIPTION
**Pull request by Sylvia23**
Warning messages are removed from the dashboard - both Measurement and Evolution Dashboard.

### CODE REVIEW

File: new-pipeline/dist.html [hunk 1]
```diff
@@ -47,16 +47,13 @@ <h2 style="font-size: 18px; margin-bottom: 0">
                     <select id="filter-os" class="multiselect" title="Target OS" data-all-selected="any OS" data-n-selected="OSs" multiple></select>
                     <select id="filter-arch" class="multiselect" title="Product architecture" data-all-selected="any architecture" data-n-selected="architectures" multiple></select>
                     with
-                    <select id="filter-e10s" class="multiselect" title="e10s setting" data-all-selected="any e10s setting" multiple></select>
```
Comment (georgf) [inline new-pipeline/dist.html]: This also looks like you included some changes from another PR here.
It's probably easiest to follow Chris instructions in #419 first.

**Review by georgf — CHANGES_REQUESTED** [review]

**Comment by Sylvia23** [comment]
@georgf Fixed that issue for this PR. Can you review now?

**Review by georgf — APPROVED** [review]
This looks good, thanks!
We should merge #419 first though.

**Comment by Sylvia23** [comment]
@georgf , Now as #419 has been merged, can we proceed for this as well. Thanks!

**Comment by georgf** [comment]
Merging this, thank you!

**Pull request merged**